### PR TITLE
chore(studio): clean slide-navigator colors

### PR DESCRIPTION
**Pull request by bloodstorms**
tucana pr => https://github.com/ToucanToco/tucana/pull/9083

### CODE REVIEW

File: small-app/nav-menu.scss [hunk 1]
```diff
@@ -80,32 +80,3 @@ nav-menu {
 .nav-menu__subchapter--new {
   color: $nav-menu-selected-color;
 }
-
-.nav-action-buttons,
-.nav-menu__add-subchapter {
-  color: $nav-menu-item-actions-color;
```
Comment (hachichaud) [inline small-app/nav-menu.scss:86]: == main color

File: small-app/nav-menu.scss [hunk 2]
```diff
@@ -80,32 +80,3 @@ nav-menu {
...
-
-  > i:hover {
-    color: $nav-menu-item-actions-hover-color;
```
Comment (hachichaud) [inline small-app/nav-menu.scss:89]: === white

File: small-app/nav-menu.scss [hunk 3]
```diff
@@ -80,32 +80,3 @@ nav-menu {
...
-  }
-}
-
-.nav-menu__add-subchapter {
-  &:hover {
-    color: $nav-menu-item-actions-hover-color;
```
Comment (hachichaud) [inline small-app/nav-menu.scss:95]: === white

File: small-app/nav-menu.scss [hunk 4]
```diff
@@ -80,32 +80,3 @@ nav-menu {
...
-  }
-}
-
-.nav-menu__chapter--active,
-.nav-menu__subchapter--active,
-.slide-navigator__unit.selected {
-  .nav-action-buttons,
-  .nav-menu__add-subchapter {
-    color: transparentize($nav-menu-item-actions-hover-color, 0.5);
```
Comment (hachichaud) [inline small-app/nav-menu.scss:104]: white opacity 0.5

File: small-app/nav-menu.scss [hunk 5]
```diff
@@ -80,32 +80,3 @@ nav-menu {
...
-  }
-  .nav-menu__add-subchapter {
-    &:hover {
-      color: $nav-menu-item-actions-hover-color;
```
Comment (hachichaud) [inline small-app/nav-menu.scss:108]: white

File: small-app/slide-navigator.scss [hunk 6]
```diff
@@ -1,3 +0,0 @@
-.slide-navigator__edit-title {
-  border-bottom: 1px solid darken($nav-menu-bg-color, 30%);
```
Comment (hachichaud) [inline small-app/slide-navigator.scss:2]: darkern(main-color-fade, 30%)

Comment (hachichaud) [reply small-app/slide-navigator.scss:2]: remplacable par text color + opacity

File: small-app/nav-menu.scss [hunk 7]
```diff
@@ -80,32 +80,3 @@ nav-menu {
 .nav-menu__subchapter--new {
   color: $nav-menu-selected-color;
 }
-
-.nav-action-buttons,
-.nav-menu__add-subchapter {
```
Comment (hachichaud) [inline small-app/nav-menu.scss:85]: styles des chapter/subchapter à laisser tel quel

**Review by ToucanTokar — APPROVED** [review]

**Pull request merged**